backend.py

```diff
@@ -1143,20 +1143,6 @@ def installBootLoader(mounts, disk, boot_partnum, primary_partnum, target_boot_m
             else:
                 installGrub2(mounts, root_partition, True)
 
-        if serial:
-            # ensure a getty will run on the serial console
```

Comment:
Doesn't this need to be reimplemented somehow instead of just dropping it?

Reply:
> Doesn't this need to be reimplemented somehow instead of just dropping it?

in xs8, we have following, So I think we can just drop it.

# inittab is no longer used when using systemd.
#
# ADDING CONFIGURATION HERE WILL HAVE NO EFFECT ON YOUR SYSTEM.
#
# Ctrl-Alt-Delete is handled by /usr/lib/systemd/system/ctrl-alt-del.target
#
# systemd uses 'targets' instead of runlevels. By default, there are two main targets:
#
# multi-user.target: analogous to runlevel 3
# graphical.target: analogous to runlevel 5
#
# To view current default target, run:
# systemctl get-default
#
# To set a default target, run:
# systemctl set-default TARGET.target
#